spikeextractors/extractors/neuroscopesortingextractor/neuroscopesortingextractor.py

```diff
@@ -26,17 +37,36 @@ class NeuroscopeSortingExtractor(SortingExtractor):
     is_writable = True
     mode = 'custom'
 
-    def __init__(self, resfile, clufile):
-        SortingExtractor.__init__(self)
-        res = np.loadtxt(resfile, dtype=np.int64, usecols=0, ndmin=1)
-        clu = np.loadtxt(clufile, dtype=np.int64, usecols=0, ndmin=1)
+    def __init__(self, resfile_path, clufile_path, keep_mua_units=True):
+        SortingExtractor.__init__(self, resfile_path, clufile_path, keep_mua_units=keep_mua_units)
```

Comment:
I think you just need to pass the self arg here. See the SortingExtractor init

Reply:
Yeah, I see that now. I'm using the subrecordingextractor as a reference due to all its extra arguments (some of which have defaults).